src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -65,6 +65,9 @@
     @Argument(fullName = StandardArgumentDefinitions.VERBOSITY_NAME, shortName = StandardArgumentDefinitions.VERBOSITY_NAME, doc = "Control verbosity of logging.", common = true, optional = true)
     public Log.LogLevel VERBOSITY = Log.LogLevel.INFO;
 
+    @Argument(fullName = "log_to_file", shortName = "log", doc = "Set the logging location", common = true, optional = true)
```

Comment:
Ah, that makes sense.  Keep them then I guess.  We can revisit them if we ever do the global naming pass.